docs/en/connector-v2/sink/TDengine.md

```diff
@@ -0,0 +1,104 @@
+# TDengine
+
+> TDengine sink connector
+
+## Description
+
+Used to write data to TDengine.
+
+## Key features
+
+- [x] [parallelism](../../concept/connector-v2-features.md)
+- [x] [support user-defined split](../../concept/connector-v2-features.md)
+
+## Options
+
+| name                       | type    | required | default value |
+|----------------------------|---------|----------|---------------|
+| url                       | string  | yes      | -             |
+| username                       | string     | yes      | -             |
+| password                  | string  | yes      | -             |
+| database                        | string  | yes      |          |
+| stable                     | string  | yes      | -             |
+| fields                   | config  | no       | -             |
+| tags                   | config  | no       | -             |
+
+### url [string] 
+
+the url of the TDengine when you select the TDengine
+
+e.g.
+```
+jdbc:TAOS-RS://localhost:6041/
+```
+
+### username [string]
+
+the username of the TDengine when you select
+
+### password [string]
+
+the password of the TDengine when you select
+
+### database [string]
+
+the database of the TDengine when you select
+
+### stable [string]
+
+the stable of the TDengine when you select
+
+### fields [string]
+
+the fields of the TDengine stable
+
+e.g.
+
+```hocon
+      fields {
+        ts = "timestamp"
+        current = "float"
+        voltage = "int"
+        phase = "float"
+        location = "string"
+        groupid = "int"
+      }
+```
```

Comment:
could you use `schema`？We uniformly use this field to describe its schema

Reply:
I refer to IotDB connector (`seatunnel-e2e/seatunnel-connector-v2-e2e/connector-iotdb-e2e/src/test/resources/iotdb/iotdb_source_to_sink.conf`)
And I think that 'fields' is the default schema name according to `org.apache.seatunnel.connectors.seatunnel.common.schema.SeaTunnelSchema#FIELD_KEY` in `org.apache.seatunnel.connectors.seatunnel.common.schema.SeaTunnelSchema#buildWithConfig`

Reply:
This config file maybe has some problems, `fields` option does not work in it. I agree with @CalvinKirs , now all connectors unified the config of schema, you can refer to https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-connectors-v2/connector-file/connector-file-local/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/local/source/LocalFileSource.java#L75 study how use `SeaTunnelSchema` get SeaTunnelRowType.

Reply:
Thank you for your advice. 
TDengine is a structured database so I can get the data schema using TDENGINE API without schema config. Therefore, I remove the schema config now.

Please give me a CR again. Thx!